todolist.html

```diff
@@ -32,5 +32,25 @@
                 <i class='icon icon--left fa fa-trash-o'></i> Delete All
             </button>
         </div>
+
+        <div>
+            <div></div> <!-- To be used for modal overlay -->
+            <div>
+                <header>
+                    <h4>Delete Task?</h4>
```

Comment:
For accessibility reasons, an h4 here doesn't make a lot of sense, especially since it is in a header element

Reply:
Removed h4 tag